### DISCUSSION TITLE
The question about DMA

### DISCUSSION THREAD

**Discussion by JingtaiChen** [stm32duino/Arduino_Core_STM32#1908] [Q&A] [answered]
Hello, I want to get a tutorial on STM32duino DMA. I wish you all the best!

**Comment by fpistm** [comment, marked as answer]
Hi  @CJT1111 

Unfortunately, I have no dedicated tutorial about DMA.
You can use all HAL and LL to achieve this.
Here an example of a STM32Cube project converted to Arduino using DMA for a F1:
https://www.stm32duino.com/viewtopic.php?t=110

It should be easy to port another project depending of your board.
